pointsource_imaging/NIRCam_Basic_PSF_Photometry_solution.ipynb

```diff
@@ -74,7 +74,7 @@
    "id": "95891849",
```

Comment:
<p>Please uncomment or delete the commented lines of code in this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
> @ojustino I would prefer to keep the commented lines in `create_psf_model()` so the users know how to modify the `psf_grid` function when they want to add the spectral source obtained using pysynphot.

In this case, I would add `source` as an argument of `create_psf_model()`.
1. I'd add `source=None` to the function definition. (`source` is eventually passed to WebbPSF's `calc_psf`, where `None` is the default value)
2. Inside `create_psf_model()`, keep the `psf = ...` lines that include `source` as an argument and delete the others.
3. Add a short docstring to the function that explains `source`, and you can move the sample call to `webbpsf.specFromSpectralType()` there.

Reply:
The latest commit implements this advice.